src/bundle/Resources/config/services.yml

```diff
@@ -390,3 +390,17 @@ services:
         parent: hautelook.router.template
         calls:
             - [ setOption, [ strict_requirements, ~ ] ]
+
+    Ibexa\Bundle\Rest\Routing\ExpressionLanguage\ContentTypeHeaderMatcherFactory:
+        arguments:
+            $mediaTypeParser: '@Ibexa\Contracts\Rest\Input\MediaTypeParserInterface'
+
+    ibexa.rest.expression_language.get_media_type:
+        class: Closure
+        factory: '@Ibexa\Bundle\Rest\Routing\ExpressionLanguage\ContentTypeHeaderMatcherFactory'
```

Comment:
```suggestion
        class: Ibexa\Bundle\Rest\Routing\ExpressionLanguage\ContentTypeHeaderMatcherFactory
```

Reply:
Simplified it to:
```yaml
    Ibexa\Bundle\Rest\Routing\ExpressionLanguage\ContentTypeHeaderMatcherFactory:
        arguments:
            $mediaTypeParser: '@Ibexa\Contracts\Rest\Input\MediaTypeParserInterface'
        tags:
            - { name: routing.expression_language_function, function: 'ibexa_get_media_type' }
```

Reply:
You will probably need a new name for the class by the way :sweat_smile: